ratis-server-api/src/main/java/org/apache/ratis/server/DivisionInfo.java

```diff
@@ -51,6 +52,10 @@ default boolean isListener() {
   /** Is this server division currently the leader and ready? */
   boolean isLeaderReady();
 
+  /** @return the current leader of this server division, null means either there is
+   * no leader for this term yet or this server division does not know who it is yet */
+  RaftPeer getCurrentLeader();
```

Comment:
Let's return RaftPeerId so that it is more efficient?  When RaftPeer is needed, we may call RaftServer.getRaftConf() to get it.
```
  /**
   * @return the id of the current leader if the leader is known to this server division;
   *         otherwise, return null.
   */
  RaftPeerId getLeaderId();
```

Reply:
Fixed